### ISSUE TITLE
Is it possible to use google-drive-ocamlfuse to mount google drive on VM instances?

### ISSUE THREAD

**Issue by Fackor** [astrada/google-drive-ocamlfuse#707]
I need to upload a large folder containing data ~20GB to google drive from a GCP VM instance. But trying to run `google-drive-ocamlfuse` for the first time gives me the following error.

```
/bin/sh: 1: xdg-open: not found
/bin/sh: 1: firefox: not found
/bin/sh: 1: google-chrome: not found
/bin/sh: 1: chromium-browser: not found
Couldn't get a file descriptor referring to the console
Cannot retrieve auth tokens.
Failure("Error opening URL
```

Is it possible currently to do what I want with this application?

**Comment by astrada** [comment]
Yes, but you have to follow the headless authorization checklist: https://github.com/astrada/google-drive-ocamlfuse/wiki/Headless-Usage-&-Authorization

**Comment by SamHobbsOrg** [comment]
I got to the page where it required consent. I am unable to use internal consent, whatever that is called. I am not sure I want to make the project available to the public.  Probably it is okay but I want to be sure.
